src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationServiceImpl registrationService
+            = new RegistrationServiceImpl();
+    private final StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+    private User actual;
+    private String login;
```

Comment:
Figure out which way is better to initialize your RegistrationServiceImpl class instance: @BeforeEach or @BeforeAll, what’s the difference?

Seems we need to use BeforeAll to initialize them all.